src/Microsoft.DocAsCode.Build.Engine/DocumentBuilder.cs

```diff
@@ -140,7 +140,7 @@ public void Build(IList<DocumentBuildParameters> parameters, string outputDirect
             var generatedManifest = ManifestUtility.MergeManifest(manifests);
 
             ManifestUtility.RemoveDuplicateOutputFiles(generatedManifest.Files);
-            _postProcessorsHandler.Handle(generatedManifest, outputDirectory);
+            _postProcessorsManager.PostProcess(generatedManifest, outputDirectory);
```

Comment:
yes, rename to process

---
In reply to: [93998741](https://github.com/dotnet/docfx/pull/1070#discussion_r93998741) [](ancestors = 93998741)